src/input/input.module.css

```diff
@@ -6,6 +8,7 @@
 }
 
 .input-group input {
+	font-family: 'Montserrat';
```

Comment:
Would it not be sufficient to add this line in the parent `.input-group`?

Reply:
Nope, the input field doesn't inherit it. 😔 I can set the font in .input-group and add font-family: inherit; to the input field, but at that point I'm adding an extra property and will end up adding 3 lines instead of 2

Reply:
Might still make it more flexible in case we change the font at some point. Or should we maybe use a variable for the font name? Anyways, we can always change that later if we want to.